test/UnitTest/Microsoft.AspNet.OData.Test.Shared/Query/Expressions/SelectExpandBinderTest.cs

```diff
@@ -304,6 +304,97 @@ public void ProjectAsWrapper_Element_ProjectedValueContainsModelID()
             Assert.Same(_model.Model, ModelContainer.GetModel(customerWrapper.ModelID));
         }
 
+        [Fact]
```

Comment:
>[Fact] [](start = 8, length = 6)

please also add (if not already present) tests for /$ref in path.  I.e., Customers(1)/Orders/$ref, Customers(1)/Orders/$ref?$filter=xyz, Customers(1)/Orders/$ref?$orderby=xyz, Customers(1)/Orders/$ref?$top=10, etc.